### PR TITLE
adds dns support for luadns

### PR DESCRIPTION
**Pull request by piratimir**

### CODE REVIEW

**Comment by piratimir** [comment]
@tonybaloney

File: libcloud/dns/drivers/luadns.py [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+from libcloud.dns.types import ZoneDoesNotExistError, ZoneAlreadyExistsError
+from libcloud.dns.types import RecordDoesNotExistError
+
+
+__all__ = [
+    'LuadnsDNSDriver'
+]
+
+
+class LuadnsDNSResponse(LuadnsResponse):
```
Comment (tonybaloney) [inline libcloud/dns/drivers/luadns.py:21]: You can just import from the common module and remove this instead

**Comment by tonybaloney** [comment]
Looks great otherwise. Let's see what the build says. You also want to add a doc page to explain the driver? You can copy existing ones and just update the logo and the website

**Comment by piratimir** [comment]
@Yes, but let's see the builds first :dancer:

**Pull request closed without merging**